README.md

```diff
@@ -1,56 +1,58 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# ECO BUILDERS
 
-> Description the project.
+> Building a construction landing page for a company that builds durable and eco-friendly buildings.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html
+- Javascript
+- CSS 
+- Behance
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[ECO BUILDERS](https://nahnahaj.github.io/construction-page/)
 
+> ![](./screenshots/app_screenshot.png)
+> ![](./screenshots/app_screenshot-mobile.png)
 
 ## Getting Started
```

Comment:
Great job @NahnahAJ 🤝

- [ ] [Optional] Just like the previous reviewer pointed out, kindly update the `Getting Started` section on the procedures to clone and run your project locally. This helps your fellow developers to know how to start reading and working on your repo.